cisco_firepower_connector.py

```diff
@@ -139,24 +139,35 @@ def _get_group_object_uuid(self):
         self.api_path = NETWORK_GROUPS_ENDPOINT.format(self.domain_uuid)
         self.debug_print("api_path: {0}".format(self.api_path))
 
-        ret_val, response = self._api_run("get", self.api_path, self)
-        if phantom.is_fail(ret_val):
-            return self.get_status()
+        offset = 0
+        limit = 50
+        params = {"limit": limit}
+        while True:
+            params["offset"] = offset
+            ret_val, response = self._api_run("get", self.api_path, self, params=params)
+            if phantom.is_fail(ret_val):
+                return self.get_status()
+            self.debug_print(f"the response is {response}")
+
+            try:
+                network_group_list = response["items"]
+                for item in network_group_list:
+                    if item["name"] == self.network_group_object:
+                        self.netgroup_uuid = item["id"]
+            except Exception as e:
+                message = "An error occurred while processing network groups"
+                self.debug_print("{}. {}".format(message, str(e)))
```

Comment:
```suggestion
                self.debug_print(f"{message}. {str(e)}")
```

should be sufficient and consistent with above uses